server/odc-core/src/main/java/com/oceanbase/odc/core/datasource/SingleConnectionDataSource.java

```diff
@@ -66,15 +71,32 @@ public class SingleConnectionDataSource extends BaseClassBasedDataSource impleme
     protected volatile Connection connection;
     private final List<ConnectionInitializer> initializerList = new LinkedList<>();
     private volatile Lock lock;
+    private final boolean keepAlive;
+    private long keepAliveIntervalMillis = 5 * 60 * 1000;
+    @Getter
+    private final String keepAliveSql = "SELECT 1 FROM DUAL";
```

Comment:
this keep live sql should be allowed to set, this datasource is not just designed for ob